busy_indicator_view.py

```diff
@@ -56,23 +56,18 @@ def on_activated_async(self, active_view):
         draw(**State)
 
 
-indicators = [
-    'Linting.  ',
-    'Linting.. ',
-    'Linting. .',
-    'Linting ..',
-    'Linting  .',
-]
+def tick():
+    while True:
+        for phase in ('.  ', '.. ', '. .' ' ..', '  .'):
+            yield "Linting" + phase
```

Comment:
In python you actually do this with `itertools.cycle`

```
phaser = cycle(['.  ', '.. ', '. .' ' ..', '  .'])
```

And then you call next(phaser)

Reply:
@kaste +1 for cycle My previous indicator was implemented exactly that way ()!
And you ripped it out. ;-)